_inc/jetpack-deactivate-dialog.js

```diff
@@ -33,19 +33,29 @@
 
 	var body = $( 'body' )[ 0 ];
 
+	var tracksProps = {
+		location: 'plugins',
+		purpose: 'deactivate',
+	};
+
 	deactivateLinkElem.attr( 'href', 'plugins.php#TB_inline?inlineId=jetpack_deactivation_dialog' );
 	deactivateLinkElem.attr( 'title', deactivate_dialog.title );
 	deactivateLinkElem.addClass( 'thickbox' );
 	deactivateLinkElem.html( deactivate_dialog.deactivate_label );
 	deactivateLinkElem.on( 'click', function( e ) {
 		observer.observe( body, { childList: true } );
+		analytics.tracks.recordEvent( 'jetpack_wpa_click', {
```

Comment:
I've never been a fan of `jetpack_wpa_click` as a top-level event name used over all. It's hard to search for a prop only. Let's update the event name to `jetpack_click_deactivate_plugin` (if that's what it's actually tracking)

Reply:
shoudl probably use the same prefix as the rest, so `jetpack_termination_dialog_something` for everything happening in this context

Reply:
changed to `jetpack_termination_dialog_open`